DetDescr/src/DetDescr/HcalGeometry.cxx

```diff
@@ -121,52 +154,47 @@ void HcalGeometry::buildStripPositionMap() {
             x = stripcenter - getZeroStrip(section, layer);
             y = 0;
           }
-	  // std::cout << "back (section,layer,strip)" << section << " " << layer << " " << strip;
-	  // std::cout << " (x,y,z) " << x << " " << y << " " << z << std::endl;
         } else {
-	  /**
-	     For side Hcal
-	     - layers in y(x)
-	     - all layers have strips in x(y) for top-bottom (left-right) sections
-	     - all layers have strips occupying width of scintillator in z (e.g. 50mm)
-	     For 3D readout:
-	     - odd layers have strips in z
-	     - even layers have strips in x(y) for top-bottom (left-right) sections
-	     - odd layers have strips occupying width of scintillator in x(y)
-	     - even layers have strips occupying width of scintillator in z
-	  */
-	  if(side_3d_readout_ && (layer%2==1)){
-	    // z position: zero-strip + half-width (center_strip) of strip
-	    z = getZeroStrip(section, layer) + getHalfTotalWidth(section, layer);
-	  }
-	  else {
-	    // z position: zero-strip(z) + strip_center(z)
-	    z = getZeroStrip(section, layer) + stripcenter;
-	  }
-	  
+          /**
+          For side Hcal
+          - layers in y(x)
+          - all layers have strips in x(y) for top-bottom (left-right) sections
+          - all layers have strips occupying width of scintillator in z (e.g.
+          50mm) For 3D readout:
+          - odd layers have strips in z
+          - even layers have strips in x(y) for top-bottom (left-right) sections
+          - odd layers have strips occupying width of scintillator in x(y)
+          - even layers have strips occupying width of scintillator in z
+          */
+          if (side_3d_readout_ && (layer % 2 == 1)) {
+            // z position: zero-strip + half-width (center_strip) of strip
+            z = getZeroStrip(section, layer) +
+                getHalfTotalWidth(section, layer);
+          } else {
+            // z position: zero-strip(z) + strip_center(z)
+            z = getZeroStrip(section, layer) + stripcenter;
+          }
+
           if (hcalsection == ldmx::HcalID::HcalSection::TOP or
               hcalsection == ldmx::HcalID::HcalSection::BOTTOM) {
-	    y = zero_layer_.at(section) + layercenter;
-	    x = getHalfTotalWidth(section, layer);
-	    if(side_3d_readout_ && layer%2==1)
-	      x = getZeroStrip(section, layer) + stripcenter;
-	    if (hcalsection == ldmx::HcalID::HcalSection::BOTTOM) {
-	      y *= -1;
-	      x *= -1;
-	    }
+            y = zero_layer_.at(section) + layercenter;
+            x = getHalfTotalWidth(section, layer);
+            if (side_3d_readout_ && layer % 2 == 1)
```

Comment:
Let's add a comment on why x is different if layer index is odd

Reply:
I changed these to use the orientation directly, so should be more clear now!